src/test/java/com/github/sdonkov/PrefixTreeTest.java

```diff
@@ -0,0 +1,80 @@
+package com.github.sdonkov;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class PrefixTreeTest {
+
+    private static List<CharSequence> words;
+    private static PrefixTree prefixTree;
+
+    @BeforeAll
+    static void setUp() throws FileNotFoundException {
+        File file = new File("src/test/resources/tests.txt");
```

Comment:
Here and below, instead of hardcoding this path, you should be able to use `getResourceAsStream("tests.txt")` now that the file is under resources. See [this thread](https://stackoverflow.com/questions/15749192/how-do-i-load-a-file-from-resource-folder).